README.md

```diff
@@ -1,19 +1,42 @@
-# rtwg_documentation
+# ROS 2 Real-Time Working Group Documentation
 
 [![Documentation Status](https://readthedocs.org/projects/real-time-working-group/badge/?version=latest)](https://real-time-working-group.readthedocs.io/en/latest/?badge=latest)
 
-## On-line documentation link
+This repository contains the sources for the ROS 2 real-time working group documentation that is hosted at https://real-time-working-group.readthedocs.io.
+The sources from this repository are built and uploaded to the site manually.
```

Comment:
There is a hook that automatically fetch and build the latest version of the master branch on every commit.